bin/fkCli.py

```diff
@@ -50,6 +50,32 @@
     "close": "stop",
     "connect": "start",
 }
+# argument should be a string
+ARGS = {
+    "path": "/path:str",
+    "data": "data:str",
+}
+
+# flag should be a set of its representations (strings)
+KWARGS = {
+    "ephemeral": {"-e", "--ephemeral"},
+    "sequence": {"-s", "--sequence"},
+    "boolean1": {"-b1", "--boolean1"},
+    "boolean2": {"-b2", "--boolean2"},
+}
+
+# program can distinguish kwarg and arg by their type
+CMD = {
+    # create /test 0x0 False False
+    "create": [ARGS["path"], ARGS["data"], KWARGS["ephemeral"], KWARGS["sequence"]],  
+    # test /test False True 0x0
+    "test": [ARGS["path"], KWARGS["boolean1"], KWARGS["boolean2"], ARGS["data"]],
```

Comment:
Since` def parse_args(cmd: str, args: List[str])` looks like have a flexibility to support other cmds like `get_data` and `set_data`, please add them.